MinPlatformPkg/Test/Library/TestPointLib/SmmTestPointLib.inf

```diff
@@ -16,10 +17,12 @@
   LIBRARY_CLASS                  = TestPointLib
 
 [Sources]
-  SmmTestPointCommunication.c
-  SmmTestPointAip.c
-  SmmTestPoint.c
+  MmTestPointCommunication.c
+  MmTestPointAip.c
+  MmTestPoint.c
+  MmTestPoint.h
   SmmTestPoint.h
+  SmmTestPointCommunication.c
```

Comment:
I guess I missed the point, but `MmTestPointCommunication.c` and `SmmTestPointCommunication.c` seems very close. Do you really need both of them in the same library? Also, do we need both of these files at all?

Reply:
You're right.  Updated.